java/1.20.5/src/schemas/WolfVariant.ts

```diff
@@ -0,0 +1,27 @@
+import {
+  StringNode as RawStringNode,
+  Mod,
+  ObjectNode,
+  Opt,
+  SchemaRegistry,
+  CollectionRegistry
+} from '@mcschema/core'
+import { Tag } from './Common'
+
+export function initWolfVariantSchema(schemas: SchemaRegistry, collections: CollectionRegistry) {
+  const StringNode = RawStringNode.bind(undefined, collections)
+
+	schemas.register('wolf_variant', Mod(ObjectNode({
+		texture: StringNode(),
+		angry_texture: StringNode(),
+		tame_texture: StringNode(),
+		biomes: Opt(Tag({ resource: '$worldgen/biome' })),
```

Comment:
Looking at the code, I don't think this field is optional, unless you tested this?